pkg/backend/identityapi/test/register.go

```diff
@@ -0,0 +1,89 @@
+package test
+
+import (
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/newrelic/infrastructure-agent/pkg/backend/identityapi"
+	"github.com/newrelic/infrastructure-agent/pkg/backend/inventoryapi"
+	"github.com/newrelic/infrastructure-agent/pkg/backend/state"
+	"github.com/newrelic/infrastructure-agent/pkg/entity"
+	"github.com/newrelic/infrastructure-agent/pkg/integrations/v4/protocol"
+)
+
+type EmptyRegisterClient struct{}
+
+func (icc *EmptyRegisterClient) RegisterEntitiesRemoveMe(agentEntityID entity.ID, entities []identityapi.RegisterEntity) (r []identityapi.RegisterEntityResponse, retryAfter time.Duration, err error) {
```

Comment:
Probably we can remove this if we don't use it anymore. Not sure don't we use it already or not.

Reply:
it's still being used for a couple tests